newapi/ooniapi/citizenlab.py

```diff
@@ -366,17 +367,17 @@ def open_pr(self, branchname):
         """Opens PR. Returns API URL e.g.
         https://api.github.com/repos/citizenlab/test-lists/pulls/800
         """
-        head = f"{self.github_user}:{branchname}"
-        log.info(f"opening a PR for {head} on {self.push_repo}")
+        head = f"{self.push_username}:{branchname}"
+        log.info(f"opening a PR for {head} on {self.origin_repo}")
```

Comment:
log.info(f"opening a PR for {head} on {self.origin_repo} using {self.push_repo}")